docs/04-operation-guides/troubleshooting/apix-04-dns-mgt-resource-ignored.md

```diff
@@ -4,7 +4,7 @@ title: External DNS management - resource ignored by the controller
 
 ## Symptom
 
-If you use a custom domain to expose a service and you have a non-Gardener clutser, the DNSProvider or DNSEntry CR you cretaed are ignored by the controller.
+If you have a non-Gardener cluster, the DNSProvider or DNSEntry CR you created are ignored by the controller.
```

Comment:
```suggestion
If you have a non-Gardener cluster, the DNS Provider or DNS Entry CR you created are ignored by the controller.
```

Reply:
^ You write it with space in other docs. Whatever option you decide on, please make it consistent in all the docs.